android/assets/jsons/translations/German.properties

```diff
@@ -3187,7 +3187,7 @@ Heal this Unit by 50 HP; Doing so will consume this opportunity to choose a Prom
 
 Accuracy I = Genauigkeit I
  # Requires translation!
-+[amount]% Strength in [param] = 
++[amount]% Strength in [param] = +[amount]% Stärke in [param]
```

Comment:
Das gibt bestimmt Kombinationen wie "+1% Stärke in Offenes Gelände"... Ungetestet & weiß keine Lösung

Reply:
Ja, solche Kombination kamen mir bereits häufig vor. Ist leider nicht so einfach wie im englischen die richtige Präposition zu finden. Zumal sie sich im Deutschen viel öfter ändern als im englischen.